File: Lesson5 Sorting/Problem2/Solutions/Solution.cpp
#include <iostream>

using namespace std;

int main(){
    int array[101], n;
    cin>>n;
    for(int i = 0; i < n; i++){
        cin>>array[i];
    }
    bool isSorted = false;
    while(!isSorted){
        isSorted = true;
        for(int i = 0; i < n - 1; i++){
            if(array[i] > array[i + 1]){
                int temp = array[i];
                array[i] = array[i + 1];
                array[i + 1] = temp;
                isSorted = false;
            }
        }
    }
    for(int i = 0; i < n; i++){
        cout<<array[i]<<endl;
    }
}

File: Lesson5 Sorting/Problem2/Solutions/Solution.py
# python 2 
array = []
n = int(raw_input())

for i in range (n):
    array.append(int(raw_input()))
isSorted = False
while isSorted == False:
    isSorted = True;
    for i in range (len(array) - 1):
        if(array[i] > array[i + 1]):
            temp = array[i]
            array[i] = array[i + 1]
            array[i + 1] = temp
            isSorted = False;

for i in array:
    print (i)
